### ISSUE TITLE
[FilterArtworks] A wildcard price range should not filter any works.

### ISSUE THREAD

**Issue by alloy** [artsy/metaphysics#575]
These example should yield identical counts.

## Without specifying a price range

```graphql
{
  gene(id: "landscape") {
    filtered_artworks(for_sale: true, aggregations: [TOTAL]) {
      total
    }
  }
}
```

```json
{
  "data": {
    "gene": {
      "filtered_artworks": {
        "total": 8752
      }
    }
  }
}
```

## With a ‘any price’ price range

```graphql
{
  gene(id: "landscape") {
    filtered_artworks(for_sale: true, aggregations: [TOTAL], price_range: "*-*") {
      total
    }
  }
}
```

```json
{
  "data": {
    "gene": {
      "filtered_artworks": {
        "total": 5433
      }
    }
  }
}
```

**Comment by alloy** [comment]
Additionally, the price range parameter could do with some documentation. E.g. is it in cents?

**Comment by alloy** [comment]
This should actually be fixed in gravity.

**Comment by cavvia** [comment]
I believe I fixed this in gravity#11140.

**Comment by alloy** [comment]
Dope 👍
